modules/electrophysiology_browser/ajax/get_electrophysiology_session_data.php

```diff
@@ -0,0 +1,326 @@
+$timePoint = \NDB_Factory::singleton()->timepoint(
+    intval($_REQUEST['sessionID'])
+);
+
+// if a user does not have the permission to view all sites' electrophsyiology
+// sessions or if a user does not have permission to view other sites' session
+// then display a Forbidden message.
+if (!$user->hasPermission('electrophysiology_browser_view_allsites')
+    && !((in_array($timePoint->getData('CenterID'), $user->getData('CenterIDs')))
+    && $user->hasPermission('electrophysiology_browser_view_site'))
+) {
+    header('HTTP/1.1 403 Forbidden');
+    exit;
+}
+
+$response = getSessionData(intval($_REQUEST['sessionID']));
+
+echo json_encode($response);
+
+/**
+ * Get the session data information.
+ *
+ * @param int $sessionID ID of the electrophysiology session
+ *
+ * @return array with the session information
+ */
+function getSessionData(int $sessionID)
+{
+    $db = \NDB_Factory::singleton()->database();
+
+    $query = 'SELECT 
+                DISTINCT(pf.SessionID) 
+              FROM physiological_file pf 
+                LEFT JOIN session s ON (s.ID=pf.SessionID) 
+                LEFT JOIN candidate c USING (CandID)
+                LEFT JOIN psc ON (s.CenterID=psc.CenterID) 
+                LEFT JOIN physiological_output_type pot 
+                  USING (PhysiologicalOutputTypeID) 
+              WHERE 
+                s.Active = "Y" 
+                AND pf.FileType IN ("bdf", "cnt", "edf", "set", "vhdr", "vsm") 
+              ORDER BY pf.SessionID';
+
+    $sessions            = $db->pselect($query, array());
+    $sessions            = array_column($sessions, 'SessionID');
+    $response['patient'] = getSubjectData($sessionID);
+    $response['database']    = array_values(getFilesData($sessionID));
+    $response['sessions']    = $sessions;
+    $currentIndex            = array_search($sessionID, $sessions);
+    $response['nextSession'] = $sessions[$currentIndex+1] ?? '';
+    $response['prevSession'] = $sessions[$currentIndex-1] ?? '';
+
+    return $response;
+}
+
+/**
+ * Get the subject data information.
+ *
+ * @param int $sessionID ID of the electrophysiology session
+ *
+ * @return array with the subject information
+ */
+function getSubjectData(int $sessionID)
+{
+    $subjectData = array();
+    $timePoint   = \NDB_Factory::singleton()->timepoint($sessionID);
+    $candidate   = \NDB_Factory::singleton()->candidate($timePoint->getCandID());
+
+    $subjectData['pscid']       = $candidate->getPSCID();
+    $subjectData['dccid']       = $timePoint->getCandID();
+    $subjectData['visit_label'] = $timePoint->getVisitLabel();
+    $subjectData['sessionID']   = $sessionID;
+    $subjectData['site']        = $timePoint->getPSC();
+    $subjectData['dob']         = $candidate->getCandidateDoB();
+    $subjectData['sex']         = $candidate->getCandidateSex();
+    $subjectData['subproject']  = $timePoint->getData('SubprojectTitle');
+    $subjectData['output_type'] = $_REQUEST['outputType'];
```

Comment:
Is this value being displayed on the front-end? If so, this value should be sanitized:

```suggestion
    $subjectData['output_type'] = htmlentities($_REQUEST['outputType'], ENT_QUOTES, "UTF-8");
```

Reply:
done